app/controllers/miq_ae_class_controller.rb

```diff
@@ -520,6 +520,7 @@ def edit_method
       id = x_node.split('-')
     end
     @ae_method = find_record_with_rbac(MiqAeMethod, id[1])
+    @embedded_methods = MiqAeMethod.where(:relative_path => @ae_method[:embedded_methods].map { |str| str.sub(/^\//, '') })
```

Comment:
`relative_path` is stored without a `/` and `embedded_methods` path are stored with `/`
```
eg : #<MiqAeMethod:0x00007fb554d37028
 id: 68655,
 name: "NotifyMessage2",
 class_id: 31261,
 display_name: "NotifyMessage2",
 description: nil,
 scope: "instance",
 language: "ruby",
 location: "inline",
 data: "[FILTERED]",
 created_on: Wed, 15 Nov 2023 09:25:10.011893000 UTC +00:00,
 updated_on: Thu, 04 Jan 2024 18:54:15.061226000 UTC +00:00,
 updated_by: "admin",
 updated_by_user_id: 1,
 embedded_methods: ["/CommonMethods/CommonNotifier/Notifier3", "/Integration/RedHat/OpenStack/DynamicDropDowns/ListFloatingIPs"],
 options: {},
 domain_id: 11785,
 relative_path: "System/Process/NotifyMessage2"> 
```

Reply:
Of note, this path does not contain the domain name.
Not sure if that meets your expectations.